CHANGELOG.md

```diff
@@ -1,15 +1,79 @@
 # Change Log
 
 ## v0.4.0
-- [plugin] added `tasks.onDidEndTask` Plug-in API
-- [plugin] Introduce `vscode.previewHtml` command support
+- [application-manager] added support for pre-load HTML templates
+- [console] added support for console `when` contexts
+- [core] added support for os `when` contexts
+- [core] added support for shell `when` contexts
+- [core] added support for vscode closure contexts
+- [core] fixed bad vertical resizing behavior
+- [core] improve scrollbar visibility for the command palette
+- [core] improve tab-bar display (display 'X' (close) on dirty editors when hovering over dirty icon)
+- [core] improve tab-bar display (display 'X' (close) only when current editor is active, or has hover)
```

Comment:
one thing i am not sure about is the verb that starts each line. should we use current or past tense? they are mixed across the log.

Reply:
thanks @elaihau I must have missed some, generally we follow the past tense.